src/PdfReader/index.tsx

```diff
@@ -400,8 +399,8 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
       navigator: {
         goForward,
         goBackward,
-        increaseFontSize,
-        decreaseFontSize,
+        zoomIn,
+        zoomOut,
         setFontFamily,
```

Comment:
let's remove setFontFamily as well, since the pdf navigator doesn't need it.

Reply:
(does the pdf navigator use setColorMode at all?)

Reply:
this can also be moved to something else, it's not important.

Reply:
Removed setFontFamily. I don't see theme change being part of the PDF TAD so I removed setColorMode as well. 

The other ones can be addressed on [SFR-1288](https://jira.nypl.org/browse/SFR-1288) I think (e.g state values)